juicy-html.html

```diff
@@ -36,14 +38,26 @@
             }
         };
         /**
+         * Skips/disregards pending request if any
+         * @return {XMLHttpRequest} skipped XHR request, in fact just `this.pending`
+         */
+        JuicyHTMLPrototype.skip = function() {
```

Comment:
`skipStampingPendingFile` is also ok for me. 

The fact that I has confused about what does this method do proves that just calling it `skip` is not enough. "Skip what?" is the question that I want to be answered by the method name.